pkg/config/config_test.go

```diff
@@ -77,18 +109,15 @@ func TestEnvOrStringVariable(t *testing.T) {
 	if i != "1" {
 		t.Errorf("Expected to get env variable, got %s insteadt", i)
 	}
+	os.Unsetenv("FOO")
 }
 
 func TestEnvOrStringDefault(t *testing.T) {
-	err := os.Unsetenv("FOO")
```

Comment:
yes you should unset it after you use it not in another test, imagine if i just ran one test I'd had that env varr hanging around

Reply:
I think this is a misconception. It's not unset here because of another test (I agree, that would be fugly), but because this test relies on the variable not being set -> otherwise it's prone to errors based on external state of env.